Backend/.gitignore

```diff
@@ -1,3 +1,4 @@
+ENV/
```

Comment:
I forgot I added that, this was before stripped out the creation of the virtualenv into runtests. Will remove